simulation/generateSearchQueries.js

```diff
@@ -0,0 +1,81 @@
+const db = require('./models/searchQueries');
+
+const DATASET_SIZE = 100000;
+const SEARCH_QUERY_ID_START = 5000000;
+const VISIT_ID_START = 1;
+const USER_ID_RANGE = 100000;
+const SEARCH_FREQUENCY_IN_SECONDS = 30;
+const MARKETS = [
+  'San Francisco', 'Seattle', 'Sydney', 'New York', 'Toronto', 'Paris',
+  'London', 'Hong Kong', 'Amsterdam', 'Montreal',
+];
+const MAX_DAYS_UNTIL_TRAVEL = 45;
+const MAX_LENGTH_OF_STAY = 7;
+const ROOM_TYPES = ['any', 'entire home', 'private', 'shared room'];
+
+/* ----- HELPER FUNCTIONS BEGIN ----- */
+
+const incrementIdFrom = (num) => {
+  let startId = num;
+  return () => {
+    startId += 1;
+    return startId;
+  };
+};
```

Comment:
OPTIONAL: Can be simplified.
`const incrementIdFrom = (num) => () => num += 1;`

Even though num is a constant, within the scope of the function you are defining it as, the value can be incremented, and it will not modify the constant itself.

Reply:
Could not apply this suggestion due to linter complaints, even after adjusting this to not return an assignment.